modules/searchbar/animated.tsx

```diff
@@ -1,9 +1,9 @@
 import * as React from 'react'
 
-import {StyleSheet, Animated} from 'react-native'
+import {StyleSheet, Animated, useWindowDimensions} from 'react-native'
 import {white} from '@frogpond/colors'
 import {SearchBar} from './searchbar'
-import {Viewport} from '@frogpond/viewport'
+import {AnimatedValueType} from './types'
 
 type Props = {
```

Comment:
Can we replace this with an import of the Props type from types.ts?

Reply:
It's not a 1-1 match at the moment, I think there was a prop missing? But I will see what I can do.